web-server/sgx-wallet-impl/src/schema/actions.rs

```diff
@@ -66,14 +67,18 @@ pub struct SignTransaction {
 
     #[zeroize(skip)]
     pub transaction_to_sign: TransactionToSign,
+
+    #[zeroize(skip)]
+    pub otp: Option<String>
```

Comment:
Like in [web-client/src/schema/actions.ts](https://github.com/ntls-io/nautilus-wallet/pull/335/files#diff-545973229f602022dfd359b6af35c03698020e5fca5d49aaa95c247b793484fb): I think we need a separate struct for `SignTransactionWithOtp`.

Reply:
We can make this just a `String` and remove the `Option` so we ensure that there has to be an OTP.